src/main/java/ua/artcode/udiary/dao/UserDaoJsonImpl.java

```diff
@@ -18,16 +19,16 @@
     // todo get path from config
     private final String DATA_PATH;
 
-    // todo GSON must be a singleton
-    private Gson gson = new GsonBuilder().setPrettyPrinting().create();
-
-    // todo init in constructor
-    private Type userListType = new TypeToken<List<User>>(){}.getType();
+    Gson gson;
```

Comment:
fields should be private. it doesn't have sense to make fiels package private in our case

Reply:
Fixed.